src/query/api/v1/handler/graphite/render.go

```diff
@@ -103,9 +103,11 @@ func (h *renderHandler) serveHTTP(
 	}
 
 	var (
-		results = make([]ts.SeriesList, len(p.Targets))
-		errorCh = make(chan error, 1)
-		mu      sync.Mutex
+		results       = make([]ts.SeriesList, len(p.Targets))
+		errorCh       = make(chan error, 1)
+		exhaustive    = true
+		exhaustiveSet bool
```

Comment:
Do you need to have this second var `exhaustiveSet` if you just set `exhaustive = true` at the beginning then always do `exhaustive = exhaustive && result.Exhaustive`? 

It should always be false if any `result.Exhaustive` is actually false and will default to true if for some reason you have zero queries.

Reply:
Sounds good